python/composio/client/utils.py

```diff
@@ -215,6 +218,9 @@ def check_cache_refresh(client: Composio) -> None:
     SDK version, and didn't come from the API. We need to start storing the data
     from the API and invalidate the cache if the data is not already stored.
     """
+    if NO_CACHE_REFRESH:
```

Comment:
Consider adding a debug log here to indicate that cache refresh is being skipped due to NO_CACHE_REFRESH being set. This would help with debugging cache-related issues.